<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GaussianFilterDialog</class>
 <widget class="QDialog" name="GaussianFilterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>166</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="lyt_k_size" stretch="0,0">
     <item>
      <widget class="QLabel" name="lbl_k_size_label">
       <property name="text">
        <string>K-Size (M, M):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spb_k_size_value">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>0</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="lyt_sigma_x" stretch="0,0">
     <item>
      <widget class="QLabel" name="lbl_sigma_x_label">
       <property name="text">
        <string>Sigma X</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spb_sigma_x_value">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>0</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="lyt_sigma_y" stretch="0,0">
     <item>
      <widget class="QLabel" name="lbl_sigma_y_label">
       <property name="text">
        <string>Sigma Y</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spb_sigma_y_value">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>0</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
